src/main/resources/application.properties

```diff
@@ -1 +1,9 @@
+spring.datasource.url=jdbc:mysql://localhost:3306/book_shop
+spring.datasource.username=root
+spring.datasource.password=Password0303
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+
+spring.jpa.show-sql=true
+spring.jpa.hibernate.ddl-auto=create-drop
+spring.jackson.deserialization.fail-on-unknown-properties=false
```

Comment:
why do you need this property?

Reply:
you didn't answer the question :) why do you need this jackson property?